### PR TITLE
Add inactive robot items and disassembly recipes, allow deactivation of all friendly robots

### PR DESCRIPTION
**Pull request by Photoloss**
#### Summary

```SUMMARY: Content "Adds inactive versions for robots only obtainable via hacking"```

#### Purpose of change

Robots hacked via control laptop can *theoretically* be dragged to new locations or attacked with impunity for subsequent disassembly. This is not very practical however and all craftable robot types can be deactivated even if that particular unit was obtained via hacking, which also makes sense as the hacking options should include a "standby mode" or "emergency shutdown".

There is no intention to add crafting recipes or disassembly yields to robots which do not have them in other contexts.

#### Describe the solution

- Adds "inactive" robot items for all robots which currently lack one, these copy the stats of the live robot or the "broken" version in that order where applicable (prices are somewhat arbitrary)
- Allows melee-range deactivation of all hacked robots by attempting to move into them, as is already the case for turrets and "hacks".
- Adds disassembly recipes for the new "inactive" robots if a "broken" version existed already. These yield the same items as the "broken" robot and take 25% longer to complete (as the broken robot likely sustained battle damage to its exterior casing and most potentially dangerous components are already inoperable)

#### Describe alternatives you've considered

- Add a direct deactivation command for the control laptop. Not implemented to keep this change JSON-only and due to lack of experience.

#### Additional context

- The tank drone's "inactive" volume was reduced to 1000 (250l) to allow transport in vehicles. This is intended to be adjusted upwards up to the full volume of the active robot as more spacious vehicle cargo components are implemented.
- Some "broken" robots have higher mass or volume than the live version. This should be corrected separately and the new value applied to the "inactive" ones.
- The deployment difficulties are somewhat arbitrary and focus on lore context rather than gameplay balance. All of these being gated behind successfully hacking with a control laptop limits the potential for abuse.
- Broken cyborgs are included as they can be hacked, but their "inactive" version cannot be butchered
- Tribots do not use their flamethrower while friendly. This is not my fault.
- I believe the "revert_to_itype" action can be used even if the hack was partially successful, theoretically "insta-killing" a dangerous foe without adequate firepower. Unless someone suggests a practical exploit I would consider the random duration combined with the need for approaching into melee range a sufficient risk.

### CODE REVIEW

**Comment by ifreund** [comment]
looks like you need to lint your json: http://dev.narc.ro/cataclysm/format.html

**Comment by Photoloss** [comment]
Uh, when I run the defense_bot.json through the linter it changes already-existing formatting, but only pasting in part of the file causes a syntax error. In this case adding the outer array brackets manually works but what's the procedure for more complex modification structures so the auto-lint does not ruin other people's deliberate deviation?

**Comment by ifreund** [comment]
looks like that file is on the [json blacklist](https://github.com/CleverRaven/Cataclysm-DDA/blob/9a010a00ec55c076cb5a37b534432d763c9be5eb/json_blacklist).  Improper formatting won't cause a build failure.  I think there was a project to lint all the json at some point but I guess it stalled.

**Comment by kevingranade** [comment]
It's still in progress, we have tooling now, but hundreds of thousands of
lines of json to style.

**Comment by FantasticFwoosh** [comment]
In such due time, players may soon be able to lift the shutters on their fortified camper van revealing the compartment of restored robots to be let loose on seek and destroy directives to clear out infested settlements ahead. I hope anyway.

**Comment by kevingranade** [comment]
Merged, had to rebase to fix conflicts.

File: data/json/monsters.json [hunk 1]
```diff
@@ -2531,6 +2540,7 @@
     "armor_bash": 12,
     "armor_cut": 8,
     "path_settings": { "max_dist": 5 },
+    "revert_to_itype": "bot_tribot",
```
Comment (ZhilkinSerg) [inline data/json/monsters.json]: There is no such item as `bot_tribot` it is called `bot_tripod`.

Comment (Photoloss) [reply data/json/monsters.json]: Thanks! Must have gotten lost at some point as I had it working locally, darn internal names.

**Pull request closed without merging**